papers/donath/donath-pylira.rst

```diff
@@ -0,0 +1,423 @@
+To obtain the most likely model given the data one searches a minimum of the total likelihood
+function, or equivalently of :math:`\mathcal{C}`. This high dimensional optimization problem
+can be solved by a classic gradient decent approach. Assuming the pixels values :math:`x_i`
+of the true image as independent parameters, one can take the derivative of the Eq. :ref:`cash`
+with respect to the individual :math:`x_i`. This way one obtains a rule for how to update the
+current set pixels :math:`\mathbf{x}_n` in each iteration of the optimization:
+
+.. math::
+   :label: rl
+
+    \mathbf{x}_{n + 1}  = \mathbf{x}_{n} -\alpha \cdot \frac{\partial \mathcal{C}\left( \mathbf{d} | \mathbf{x} \right)}{\partial x_i}
+
+Where :math:`\alpha` is a factor to define the step size. It was shown by :cite:`Richardson1972`
+that this converges to a maximum likelihood solution of Eq. :ref:`cash`. This method
+is in general equivalent to the gradient decent and backpropagation methods used in
+modern machine learning techniques. A Python implementation of the standard RL method
+is available e.g. in the `Scikit-Image` package :cite:`skimage`. Instead of the gradient
+decent based optimization it is also possible to sample from the likelihood function using
+a simple Metropolis-Hastings approach. This is demonstrated in one of the *Pylira* online
+tutorials (`Introduction to Deconvolution using MCMC Methods <https://pylira.readthedocs.io/en/latest/pylira/user/tutorials/notebooks/mcmc-deconvolution-intro.html>`__).
+
+While technically the RL method converges to a maximum likelihood solution, it mostly
+still results in poorly restored images, especially if extended emission regions are
+present in the image. Because of the PSF convolution an extended emission region
+can decompose into multiple nearby point sources and still lead to good model prediction,
+when compared with the data. Those almost equally good solutions correspond
+to many narrow local minima or "spikes" in the global likelihood surface. Depending
+on the start estimate for the reconstructed image :math:`\mathbf{x}` the RL method will follow
+the steepest gradient and converge towards the nearest narrow local minimum.
+This problem has been described by multiple authors, such as :cite:`Reeves1994`
+and :cite:`Fish95`.
+
+
+The LIRA Multiscale Prior
++++++++++++++++++++++++++
+One possible solution to this problem was described in :cite:`Esch2004`.
+First the standard RL method can be extended by taking into account
+the non uniform exposure :math:`e_i` and a background estimate :math:`b_i`:
+
+.. math::
+   :label: convolution
+
+    \lambda_i = \sum_k (e_i \cdot x_i) p_{i - k} + b_i
+
+Second the authors proposed to extend the Poisson log-likelihood
+function (Eq. :ref:`cash`) by a log-prior term that controls the
+smoothness of the reconstructed
+image on multiple spatial scales. For this the image is transformed
```

Comment:
Perhaps here you can include a version of Figure 2 from Esch et al paper to illustrate the levels?

Reply:
Done